kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/shops/GlutenFreeShop.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.good.patterns.food2door.shops;
+
+import com.kodilla.good.patterns.food2door.*;
+import com.kodilla.good.patterns.food2door.interfaces.InformationService;
+import com.kodilla.good.patterns.food2door.interfaces.Manufacturer;
+import com.kodilla.good.patterns.food2door.interfaces.OrderService;
+import com.kodilla.good.patterns.food2door.simple.classes.Product;
+import com.kodilla.good.patterns.food2door.simple.classes.Supplier;
+
+public class GlutenFreeShop implements Manufacturer, InformationService, OrderService {
```

Comment:
S z SOLID to Single Responsibility Pronciple (zasada pojedynczej odpowiedzialnosci).
Jaką odpowiedzialność ma producent żywności? Wyprodukować (process) żywność

Reply:
public class GlutenFreeShop implements Manufacturer { .... }
a Manufacturer posiada dwie metody: 
* public String getName()
* public boolean process(String product, double quantity)

Reply:
Cloneable --> clone()
Comparable --> compareTo() ....